.circleci/config.yml

```
@@ -117,9 +117,12 @@ workflows:
     jobs:
       - python-flake8-tests
       - legacy-python-flake8-tests
-      - backend-unit-tests
       - frontend-unit-tests
       - frontend-e2e-tests
+      - backend-unit-tests:
+          filters:
+            tags:
+              only: /.*/
```

I think we'll want to do this here since our tags have the `m` prefix?
```suggestion
              only: /m.*/
```

---

I've updated the tag to `only: /^m[0-9]+(\.[0-9]+)?$/` which is the same as in `deploy-milestone`